packages/accordion/src/AccordionDetails.tsx

```diff
@@ -0,0 +1,46 @@
+import { ReactNode, useEffect, useRef, useState } from 'react'
+
+import { Content, ContentWrapper } from './styled'
+
+interface AccordionDetailsProps {
+  children?: ReactNode
+  isExpanded?: boolean
+  className?: string
+}
+
+export function AccordionDetails({
+  children,
+  isExpanded = false,
+  className,
+}: AccordionDetailsProps) {
+  const resizeObserver = useRef<ResizeObserver>()
```

Comment:
Optional: Can we group all the logic into a `useController` hook?

```
const self = useController()
```

Reply:
I think the current logic is simple so we don't need to group all the logic into a useController.
Also it may bring confusion in this simple function.